index.css

```diff
@@ -388,3 +403,213 @@ label {
   font-weight: 400;
   line-height: 28px;
 }
+
+.list_style {
+  list-style: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .nav2 {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  .section1 h1 {
+    font-size: 48px;
+    color: #172b4d;
+    line-height: 60px;
+    font-weight: 700;
+    width: 420px;
+  }
+
+  .section1 p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 28px;
+    width: 580px;
+    font-weight: 400;
+    align-items: left;
+    padding: 50px 0 0 0;
+  }
```

Comment:
Hello @EmekaEmmanuel, I've noticed that you are using the wrong typography for a lot of elements in your website. Please notice that the desktop version requires a different scale for your elements 😉. But don't worry, you can always check for the whole typography in Figma. To do so, just hover over any element and click on it, a window will be displayed with the correct properties values:
![image](https://user-images.githubusercontent.com/42799579/199865260-027e9be8-a5e4-4cdc-ae19-65f43478a5fb.png)

I've checked the following elements, please apply the typography display in Figma to them:
- .section paragraph:
![image](https://user-images.githubusercontent.com/42799579/199865392-99886d07-a8dc-41bd-85a6-26b3b0a6985c.png)
- .articlediv paragraph:
![image](https://user-images.githubusercontent.com/42799579/199865514-23c73251-5984-4592-b712-b3dbf4c4ab3b.png)
- .about_self_section paragraph:
![image](https://user-images.githubusercontent.com/42799579/199865735-ee612b59-ee77-41c3-8780-69593b384b74.png)
- languages text:
![image](https://user-images.githubusercontent.com/42799579/199865817-d77580db-c7e4-446c-90d4-923590eda8be.png)
- .contact-form paragraph:
![image](https://user-images.githubusercontent.com/42799579/199865888-c187b668-6385-466c-8067-d0e17932b96a.png)

If you have any doubts please tag me 😄.

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.